backend/app/api/src/endpoints/global/members/PatchOrganizationMembersEndpoint.ts

```diff
@@ -306,7 +306,7 @@ export class PatchOrganizationMembersEndpoint extends Endpoint<Params, Query, Bo
                         }
                     }
                     else {
-                        if (registration.periodId !== platform.periodId) {
+                        if (registration.periodId !== platform.periodIdIfPlatform) {
```

Comment:
Dit werkt niet en geeft een error. Snap het niet echt.

Zou beter vervangen woorden door iets slimmer dat de organization period id teruggeeft in organization mode, en die van de platform in platform mode.